### ISSUE TITLE
Please add line login api

### ISSUE THREAD

**Issue by puzzlethai** [MrSwitch/hello.js#499]
I try to use line login api with client-side only web app but It's don't work because of CORS. Is it possible that hello.js can solve the problem?  FYI On https://developers.line.me/web-api/integrating-web-login-v2 told that their feature is based on the OAuth 2.0 protocol.

**Comment by MrSwitch** [comment]
Yes, hellojs supports that but you'll have to contribute the module.

**Comment by h4i4nhnc** [comment]
@MrSwitch
I add line modules here https://github.com/MrSwitch/hello.js/pull/653
